static/js/notifications.js

```diff
@@ -30,7 +30,9 @@ if (window.webkitNotifications) {
                 return 2;
             }
         },
-        requestPermission: window.Notification.requestPermission,
+        requestPermission: function () {
+            window.Notification.requestPermission();
```

Comment:
Hmm why doesn't this take a callback argument?  It seems like you're passing a callback in the code below...